packages/nouns-webapp/src/components/StandaloneNoun/index.tsx

```diff
@@ -47,17 +61,28 @@ export const StandaloneNounWithSeed: React.FC<StandaloneNounWithSeedProps> = (
 ) => {
   const { nounId, onLoadSeed, shouldLinkToProfile } = props;
 
+  const dispatch = useDispatch();
+  const history = useHistory();
   const seed = useNounSeed(nounId);
 
   if (!seed || !nounId || !onLoadSeed) return <Noun imgPath="" alt="Noun" />;
 
   onLoadSeed(seed);
 
+  const onClickHandler = () => {
+    dispatch(setOnDisplayAuctionNounId(nounId.toNumber()));
+    history.push(`/noun/${nounId}`);
```

Comment:
I don't think this line is necessary given the `to` parameter in the `Link` comp